checks.d/snmp.py

```diff
@@ -238,6 +238,8 @@ def check(self, instance):
         ip_address = instance["ip_address"]
         table_oids = []
         raw_oids = []
+        timeout = instance.get('timeout', self.DEFAULT_TIMEOUT)
+        retries = instance.get('retries', self.DEFAULT_RETRIES)
```

Comment:
And not your code but same here, https://github.com/DataDog/dd-agent/pull/1417/files#diff-b7a49ef3eaa09d1278637a175c5642cbR127 ?